biomaj_core/utils.py

```diff
@@ -460,9 +462,13 @@ def uncompress(archivefile, remove=True):
             elif archivefile.endswith('.zip'):
                 subprocess.check_call("unzip -o " + archivefile + " -d " + os.path.dirname(archivefile), shell=True)
                 is_archive = True
+            else:
+                logger.warn('Uncompress: Unknown file extension')
         except CalledProcessError as uncompresserror:
             logger.error("Uncompress error of %s: %s" % (archivefile, str(uncompresserror)))
             return False
+        except Exception as e:
```

Comment:
Which exception do you try to catch?
Should also return False in case of error

Reply:
I tried to catch error when extension is not part of the ones listed in the conditional statement. Up to now, downloaded files with unknown extension remained compressed without raising error.